docs/user/upgrade.rst

```diff
@@ -37,6 +37,25 @@ Packages
 
 Use your systems package management.
 
+Docker
+^^^^^^
+
+**ATTENTION** Currently docker version can differ from intelmq version namings.
+
+.. code-block:: bash
+
+   docker pull cerat/intelmq-full:1.0
+
+You can check the current versions from intelmq & intelmq-manager via git commit ref.
+
+The first one is the intelmq commit ref, and the second one is for intelmq-manager
+
```

Comment:
If these two sentences belong together I would remove the empty line in-between. If the second statement refers to the code-block below I would add a `:` at the end to make that clear.